test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -48,10 +53,18 @@ public CCMBridge(Config config)
 
     public CCMBridge(int nodeCount)
     {
+        this.nodeCount = nodeCount;
         this.ccmDir = Files.createTempDir();
         execute("ccm create %s -n %d --install-dir %s", DEFAULT_CLUSTER_NAME, nodeCount, CASSANDRA_DIR);
     }
 
+    public void captureLogs(String testName)
+    {
+        stop();
```

Comment:
Again, I feel this is mixing concerns. You're violating Single Responsibility: https://en.wikipedia.org/wiki/SOLID_%28object-oriented_design%29 If you want the logs captured after a stop call, then you should be calling stop() prior to calling captureLogs() in the tearDown() of harnessTest.